src/providers/eth_provider/provider.rs

```diff
@@ -328,18 +330,26 @@ where
             let mut nonce = DEPLOY_WALLET_NONCE.lock().await;
             let current_nonce = *nonce;
 
-            let tx = execution
+            // let tx = execution
+            //     .nonce(current_nonce)
+            //     .max_fee(u64::MAX.into())
+            //     .prepared()
+            //     .map_err(|_| EthApiError::EthereumDataFormat(EthereumDataFormatError::TransactionConversion))?
+            //     .get_invoke_request(false)
+            //     .await
+            //     .map_err(|_| SignatureError::SigningFailure)?;
+            // self.starknet_provider
+            //     .add_invoke_transaction(BroadcastedInvokeTransaction::V1(tx))
+            //     .await
+            //     .map_err(KakarotError::from)?;
```

Comment:
should we uncomment this?

Reply:
In fact we can just remove all this, just did it because this is fully replaced by

```rust
let prepared_execution = execution
                .nonce(current_nonce)
                .max_fee(u64::MAX.into())
                .prepared()
                .map_err(|_| EthApiError::EthereumDataFormat(EthereumDataFormatError::TransactionConversion))?;
```

`get_invoke_request` is now not accessible and it is private.